examples/rapids/02-rapids-gpu-cluster.ipynb

```diff
@@ -0,0 +1,542 @@
+   "source": [
+    "<br>\n",
+    "<details><summary>If you want to learn more about how this function works, click here. </summary>\n",
+    "    \n",
+    "* **Creates a target column** - The raw data does not contain a column that cleanly describes whether a particular ride could be considered a \"good\" tip. So we need to add one! In this exercise, a \"good\" tip is one that is greater than 20% of the fare amount. This function makes this a binary classification problem.\n",
+    "* **Adds additional features** - Here, instead of giving a model a raw timestamp, we provide multiple derived characteristics like hour of the day and day of the week. Giving the machine learning model a richer description of each training observation improves its ability to describe the relationship between those observations' characteristics and the target.\n",
+    "* **Removes Unused Columns** - If the goal is to produce a model that could predict whether a tip will be good *at the beginning of the trip*, then characteristics that can only be known **AFTER** the tip must be excluded. For example, you can't know the dropoff time or the type of payment until a ride has concluded.\n",
+    "* **Cast the Columns to 32-bit types** - This reduces the memory footprint of the model and insures compatablility with older version of `cuml`.\n",
+    "    \n",
+    "</details>\n",
+    "<br> \n",
+    "We now just need to run the raw data through our function to prepare it for machine learning."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "taxi = prep_df(taxi)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "Since this is a binary classification task, before proceeding we should examine the proportion of 1s and 0s in the target. This can be done with the `value_counts()` method.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "taxi[\"target\"].value_counts(normalize=True).compute()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that the dataframe has been processed, let's check its length and size in memory again."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_rows = len(taxi)\n",
+    "memory_usage = taxi.memory_usage(deep=True).sum().compute() / 1e9\n",
+    "print(f\"Num rows: {num_rows}, Memory Usage: {memory_usage} GB\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Removing unused columns dropped the size of the training data to about one-third the size of the raw data. You can also see that the dataset lost a few rows with zero fare amounts."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Train a Random Forest Model\n",
```

Comment:
I would cut most of the text from here to the line `taxi_test = prep_df(taxi_test)`, since it's all already in the first example. I would would say a sentence of "_this code is all the exact same_ (except we load the test data into a `dask_cudf`)"